### PR TITLE
Add more localization on setup tab

### PR DESCRIPTION
**Pull request by HThuren**
Localization of on setup tab
image

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

File: src/tabs/setup.html [hunk 1]
```diff
@@ -75,11 +75,11 @@
                         <canvas id="canvas"></canvas>
                         <div class="attitude_info">
                             <dl>
-                                <dt>Heading:</dt>
-                                <dd class="heading">&nbsp;</dd>
-                                <dt>Pitch:</dt>
+                                <dt i18n="initialSetupHeading"></dt>
+                                <dd class="heading" width>&nbsp;</dd>
```
Comment (McGiverGim) [inline src/tabs/setup.html:79]: What is this width `attribute` for?

File: locales/en/messages.json [hunk 2]
```diff
@@ -822,6 +822,15 @@
...
     },
+    "initialSetupHeading": {
+        "message": "Heading:"
+    },
+    "initialSetupPitch": {
+        "message": "Pitch:"
+    },
+    "initialSetupRoll": {
+        "message": "Roll:"
+    },
```
Comment (McGiverGim) [inline locales/en/messages.json:833]: I always recommend to add a description to let know the translators where the strings are used.

File: src/tabs/setup.html (hunk 1 above)
Comment (HThuren) [reply src/tabs/setup.html]: Type error

**Review by haslinghuis — APPROVED** [review]
Please change title of the PR

**Review by nerdCopter — APPROVED** [review]

**Comment by HThuren** [comment]
@haslinghuis @nerdCopter - thank you

**Pull request merged**